contrib/zlib/module.defs

```diff
@@ -1,9 +1,9 @@
 $(eval $(call import.MODULE.defs,ZLIB,zlib))
 $(eval $(call import.CONTRIB.defs,ZLIB))
 
-ZLIB.FETCH.url     = https://github.com/HandBrake/HandBrake-contribs/releases/download/contribs/zlib-1.2.12.tar.gz
```

Comment:
This shouldn't be removed, but updated with the current file name, then we'll upload it to the contribs repository.